machines-endpoints/TAG

```diff
@@ -1 +1 @@
-0.4.0
```

Comment:
Just a comment; this is to update k8s client modules.
If we had updated the modules, machines-endpoints need not be updated because its image is scratch-based.